stable-version.sbt

```diff
@@ -0,0 +1 @@
+version in ThisBuild := "0.1.2"
```

Comment:
nit: if we're not reading this from sbt, might be easier to just make this a text file (like the content is only `0.1.2`).